### ISSUE TITLE
Remove update cron job

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#412]
Now that we have #396 merged, once #389 or some improved version thereof is also merged (to ensure users reboot their workstations after some time), we can remove the daily cron job which runs `securedrop-update`.

Also blocked on #427.

**Comment by eloquence** [comment]
Q: Are there reasons to keep `securedrop-update` itself around at that point?

**Comment by eloquence** [comment]
> Q: Are there reasons to keep securedrop-update itself around at that point?

Per discussion at sprint planning today we can safely remove it as part of this change; we can always dig it up alter if needed.

**Comment by conorsch** [comment]
There's a crucial line in the cron job that enforces salt state:

https://github.com/freedomofpress/securedrop-workstation/blob/08a064e1d6d0a015712a2068c8b202594e760553/dom0/securedrop-update#L48

The rest of the cron logic can be culled, but we need some mechanism to enforce salt logic over time, via automatic updates. Perhaps we'll want to fold that into the launcher logic added via #396, or somewhere else. @redshiftzero @eloquence if you feel so inclined, happy to port discussion of salt state to another ticket.

**Comment by eloquence** [comment]
Very clear, thanks. Added to discussion points; let's discuss strategies for enforcing this state. Of course, if this is a consistently long-running operation it would be ideal not to have the user go through it on every launch, or even every launch that installs some unrelated package updates.
